Description: change to pass hardening flags
Author: Nicolas HOUDELOT (nicolas@demosdebs.org)
Last-Update: 2020-04-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/build/Makefile
+++ b/build/Makefile
@@ -40,6 +40,9 @@
 LDFLAGS += -lX11 -lXext -lXfixes
 endif
 
+CFLAGS+=$(shell dpkg-buildflags --get CFLAGS) $(shell dpkg-buildflags --get CPPFLAGS)
+LDFLAGS+=$(shell dpkg-buildflags --get LDFLAGS)
+
 EXEC=hxcfloppyemulator
 
 FTDIFLOPPYEMU=usb_hxcfloppyemulator.o variablebitrate.o
